### PR TITLE
Add option to use numerical hessian approximation for contribution of external cost module

### PR DESCRIPTION
**Pull request by FreyJo**
add option to use custom hessian approximation for external cost module

### CODE REVIEW

File: acados/ocp_nlp/ocp_nlp_cost_external.c [hunk 1]
```diff
@@ -217,6 +219,15 @@ int ocp_nlp_cost_external_model_set(void *config_, void *dims_, void *model_,
     {
         model->ext_cost_fun_jac_hess = (external_function_generic *) value_;
     }
+    else if (!strcmp(field, "ext_cost_fun_jac"))
+    {
+        model->ext_cost_fun_jac = (external_function_generic *) value_;
+    }
+    else if (!strcmp(field, "cost_custom_hess"))
```
Comment (giaf) [inline acados/ocp_nlp/ocp_nlp_cost_external.c:226]: maybe `ext_cost_custom_hess` is more in line with the other namings

Comment (giaf) [reply acados/ocp_nlp/ocp_nlp_cost_external.c:226]: or even maybe just 'ext_cost_hess'

Comment (FreyJo) [reply acados/ocp_nlp/ocp_nlp_cost_external.c]: I think I renamed all the fields as we discussed.

Comment (giaf) [reply acados/ocp_nlp/ocp_nlp_cost_external.c]: yes that was before our discussion

**Pull request merged**